jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessageUtils.kt

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs.toolwindow
+
+import com.intellij.util.ui.ColumnInfo
+import org.apache.commons.lang.StringUtils
+import software.amazon.awssdk.services.sqs.model.Message
+import software.amazon.awssdk.services.sqs.model.MessageSystemAttributeName
+import software.aws.toolkits.jetbrains.utils.ui.ResizingColumnRenderer
+import software.aws.toolkits.jetbrains.utils.ui.WrappingCellRenderer
+import software.aws.toolkits.resources.message
+import java.lang.IllegalArgumentException
+import javax.swing.table.TableCellRenderer
+
+class MessageIdColumn : ColumnInfo<Message, String>(message("sqs.message.message_id")) {
+    private val renderer = ResizingColumnRenderer()
+    override fun valueOf(item: Message?): String? = item?.messageId()
+    override fun isCellEditable(item: Message?): Boolean = false
+    override fun getRenderer(item: Message?): TableCellRenderer? = renderer
+}
+
+class MessageBodyColumn : ColumnInfo<Message, String>(message("sqs.message.message_body")) {
+    private val renderer = WrappingCellRenderer(wrapOnSelection = true, toggleableWrap = false)
+    override fun valueOf(item: Message?): String? = StringUtils.abbreviate(item?.body(), MAX_LENGTH_OF_MESSAGES)
+    override fun isCellEditable(item: Message?): Boolean = false
+    override fun getRenderer(item: Message?): TableCellRenderer? = renderer
+
+    private companion object {
+        // Truncated the message body to show up to 1KB, as it can be up to 256KB in size. Cannot limit the retrieved message size through API.
+        const val MAX_LENGTH_OF_MESSAGES = 1024
```

Comment:
Would prefer this in a more central location like `SQSUtils.kt` or something along those lines?

Reply:
Sounds like a good solution, I will create a new SqsUtils file.